scripts/saveState.js

```diff
@@ -66,8 +66,7 @@ function addToSandwich(type, item){
         }
 		sidebarBody().find("#sideSubgoal" + item.id).unbind( "click" ).click(function(){
 			// do not enter drawSubgoal with a different id until the current subgoal is saved
-			var isSetSubgoalQuestions = (statusIsTrue("gotSubgoalQuestions"));
-			if (isSetSubgoalQuestions || item.id == subArr.lengh){
+			if (statusIsTrue("gotSubgoalQuestions")){
```

Comment:
The current (last) subgoal is already being displayed, so there isn't a need for the second condition. As long as the current subgoal is saved, it is safe to click on the others.

Reply:
👍

Reply:
Great catch